keps/sig-network/4444-service-traffic-distribution/kep.yaml

```diff
@@ -20,21 +20,22 @@ see-also:
   - "https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/2086-service-internal-traffic-policy"
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: alpha
+stage: beta
 
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.30"
+latest-milestone: "v1.31"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.30"
+  # beta: "v1.31"
```

Comment:
I think you want to uncomment this?

Reply:
Ah thanks for catching that! Uncommented.